dockerFiles/marklogic-server-centos:base

```diff
@@ -111,3 +111,8 @@ VOLUME /var/opt/MarkLogic
 ###############################################################
 ENTRYPOINT ["/usr/local/bin/start-marklogic.sh"]
 
+###############################################################
+# remove mlcmd packages
+###############################################################
+RUN sudo rm -rf ./opt/MarkLogic/mlcmd/lib/*
```

Comment:
Please move this up to the line with "RUN yum localinstall -y /tmp/marklogic-server.rpm" as it should be part of the same layer and logically, it is part of the cleanup of the install of the RPM. Use "&&" syntax to run multiple commands in one RUN directive.

Reply:
I agree with moving it up to the same spot and use && syntax instead of two separate runs but the "install MarkLogic server and sudo" comment above the RUN command you are recommending me adding the rm's to doesn't cover what the remove does so I still feel it should be a separate comment block/RUN statement

Reply:
We should keep this in one RUN command along with all of the other things that are done to install the RPM. Update the comment to include details about why you are removing the MLCMD libs. We do not need to create another container layer just for this command.